FYI, netlib is also broken on Windows since Windows build does not have `-lblas` or `-lcblas`. Is this intended and my problem?

```
gonum.org/v1/netlib/blas/netlib
# gonum.org/v1/netlib/blas/netlib
c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_caxpy':

/tmp/go-build/cgo-gcc-prolog:57: undefined reference to `cblas_caxpy'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ccopy':

/tmp/go-build/cgo-gcc-prolog:76: undefined reference to `cblas_ccopy'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cdotc_sub':

/tmp/go-build/cgo-gcc-prolog:96: undefined reference to `cblas_cdotc_sub'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cdotu_sub':

/tmp/go-build/cgo-gcc-prolog:116: undefined reference to `cblas_cdotu_sub'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cgbmv':

/tmp/go-build/cgo-gcc-prolog:144: undefined reference to `cblas_cgbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cgemm':

/tmp/go-build/cgo-gcc-prolog:172: undefined reference to `cblas_cgemm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cgemv':

/tmp/go-build/cgo-gcc-prolog:198: undefined reference to `cblas_cgemv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cgerc':

/tmp/go-build/cgo-gcc-prolog:223: undefined reference to `cblas_cgerc'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cgeru':

/tmp/go-build/cgo-gcc-prolog:248: undefined reference to `cblas_cgeru'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_chbmv':

/tmp/go-build/cgo-gcc-prolog:274: undefined reference to `cblas_chbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_chemm':

/tmp/go-build/cgo-gcc-prolog:302: undefined reference to `cblas_chemm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_chemv':

/tmp/go-build/cgo-gcc-prolog:328: undefined reference to `cblas_chemv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cher':

/tmp/go-build/cgo-gcc-prolog:349: undefined reference to `cblas_cher'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cher2':

/tmp/go-build/cgo-gcc-prolog:374: undefined reference to `cblas_cher2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cher2k':

/tmp/go-build/cgo-gcc-prolog:401: undefined reference to `cblas_cher2k'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cherk':

/tmp/go-build/cgo-gcc-prolog:424: undefined reference to `cblas_cherk'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_chpmv':

/tmp/go-build/cgo-gcc-prolog:448: undefined reference to `cblas_chpmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_chpr':

/tmp/go-build/cgo-gcc-prolog:467: undefined reference to `cblas_chpr'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_chpr2':

/tmp/go-build/cgo-gcc-prolog:490: undefined reference to `cblas_chpr2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cswap':

/tmp/go-build/cgo-gcc-prolog:542: undefined reference to `cblas_cswap'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_csymm':

/tmp/go-build/cgo-gcc-prolog:570: undefined reference to `cblas_csymm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_csyr2k':

/tmp/go-build/cgo-gcc-prolog:598: undefined reference to `cblas_csyr2k'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_csyrk':

/tmp/go-build/cgo-gcc-prolog:623: undefined reference to `cblas_csyrk'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ctbmv':

/tmp/go-build/cgo-gcc-prolog:646: undefined reference to `cblas_ctbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ctbsv':

/tmp/go-build/cgo-gcc-prolog:669: undefined reference to `cblas_ctbsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ctpmv':

/tmp/go-build/cgo-gcc-prolog:690: undefined reference to `cblas_ctpmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ctpsv':

/tmp/go-build/cgo-gcc-prolog:711: undefined reference to `cblas_ctpsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ctrmm':

/tmp/go-build/cgo-gcc-prolog:737: undefined reference to `cblas_ctrmm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ctrmv':

/tmp/go-build/cgo-gcc-prolog:760: undefined reference to `cblas_ctrmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ctrsm':

/tmp/go-build/cgo-gcc-prolog:786: undefined reference to `cblas_ctrsm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ctrsv':

/tmp/go-build/cgo-gcc-prolog:809: undefined reference to `cblas_ctrsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dasum':

/tmp/go-build/cgo-gcc-prolog:828: undefined reference to `cblas_dasum'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_daxpy':

/tmp/go-build/cgo-gcc-prolog:851: undefined reference to `cblas_daxpy'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dcopy':

/tmp/go-build/cgo-gcc-prolog:870: undefined reference to `cblas_dcopy'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ddot':

/tmp/go-build/cgo-gcc-prolog:892: undefined reference to `cblas_ddot'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dgbmv':

/tmp/go-build/cgo-gcc-prolog:923: undefined reference to `cblas_dgbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dgemm':

/tmp/go-build/cgo-gcc-prolog:951: undefined reference to `cblas_dgemm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dgemv':

/tmp/go-build/cgo-gcc-prolog:977: undefined reference to `cblas_dgemv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dger':

/tmp/go-build/cgo-gcc-prolog:1002: undefined reference to `cblas_dger'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dnrm2':

/tmp/go-build/cgo-gcc-prolog:1021: undefined reference to `cblas_dnrm2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_drot':

/tmp/go-build/cgo-gcc-prolog:1045: undefined reference to `cblas_drot'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_drotm':

/tmp/go-build/cgo-gcc-prolog:1080: undefined reference to `cblas_drotm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_drotmg':

/tmp/go-build/cgo-gcc-prolog:1096: undefined reference to `cblas_drotmg'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dsbmv':

/tmp/go-build/cgo-gcc-prolog:1122: undefined reference to `cblas_dsbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dsdot':

/tmp/go-build/cgo-gcc-prolog:1161: undefined reference to `cblas_dsdot'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dspmv':

/tmp/go-build/cgo-gcc-prolog:1188: undefined reference to `cblas_dspmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dspr':

/tmp/go-build/cgo-gcc-prolog:1208: undefined reference to `cblas_dspr'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dspr2':

/tmp/go-build/cgo-gcc-prolog:1231: undefined reference to `cblas_dspr2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dswap':

/tmp/go-build/cgo-gcc-prolog:1250: undefined reference to `cblas_dswap'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dsymm':

/tmp/go-build/cgo-gcc-prolog:1278: undefined reference to `cblas_dsymm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dsymv':

/tmp/go-build/cgo-gcc-prolog:1304: undefined reference to `cblas_dsymv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dsyr':

/tmp/go-build/cgo-gcc-prolog:1326: undefined reference to `cblas_dsyr'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dsyr2':

/tmp/go-build/cgo-gcc-prolog:1351: undefined reference to `cblas_dsyr2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dsyr2k':

/tmp/go-build/cgo-gcc-prolog:1379: undefined reference to `cblas_dsyr2k'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dsyrk':

/tmp/go-build/cgo-gcc-prolog:1404: undefined reference to `cblas_dsyrk'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dtbmv':

/tmp/go-build/cgo-gcc-prolog:1427: undefined reference to `cblas_dtbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dtbsv':

/tmp/go-build/cgo-gcc-prolog:1450: undefined reference to `cblas_dtbsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dtpmv':

/tmp/go-build/cgo-gcc-prolog:1471: undefined reference to `cblas_dtpmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dtpsv':

/tmp/go-build/cgo-gcc-prolog:1492: undefined reference to `cblas_dtpsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dtrmm':

/tmp/go-build/cgo-gcc-prolog:1518: undefined reference to `cblas_dtrmm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dtrmv':

/tmp/go-build/cgo-gcc-prolog:1541: undefined reference to `cblas_dtrmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dtrsm':

/tmp/go-build/cgo-gcc-prolog:1567: undefined reference to `cblas_dtrsm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dtrsv':

/tmp/go-build/cgo-gcc-prolog:1590: undefined reference to `cblas_dtrsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dzasum':

/tmp/go-build/cgo-gcc-prolog:1609: undefined reference to `cblas_dzasum'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dznrm2':

/tmp/go-build/cgo-gcc-prolog:1631: undefined reference to `cblas_dznrm2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_icamax':

/tmp/go-build/cgo-gcc-prolog:1654: undefined reference to `cblas_icamax'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_idamax':

/tmp/go-build/cgo-gcc-prolog:1677: undefined reference to `cblas_idamax'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_isamax':

/tmp/go-build/cgo-gcc-prolog:1700: undefined reference to `cblas_isamax'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_izamax':

/tmp/go-build/cgo-gcc-prolog:1723: undefined reference to `cblas_izamax'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sasum':

/tmp/go-build/cgo-gcc-prolog:1746: undefined reference to `cblas_sasum'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_saxpy':

/tmp/go-build/cgo-gcc-prolog:1768: undefined reference to `cblas_saxpy'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_scasum':

/tmp/go-build/cgo-gcc-prolog:1788: undefined reference to `cblas_scasum'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_scnrm2':

/tmp/go-build/cgo-gcc-prolog:1811: undefined reference to `cblas_scnrm2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_scopy':

/tmp/go-build/cgo-gcc-prolog:1833: undefined reference to `cblas_scopy'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sdot':

/tmp/go-build/cgo-gcc-prolog:1856: undefined reference to `cblas_sdot'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sdsdot':

/tmp/go-build/cgo-gcc-prolog:1882: undefined reference to `cblas_sdsdot'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sgbmv':

/tmp/go-build/cgo-gcc-prolog:1913: undefined reference to `cblas_sgbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sgemm':

/tmp/go-build/cgo-gcc-prolog:1941: undefined reference to `cblas_sgemm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sgemv':

/tmp/go-build/cgo-gcc-prolog:1967: undefined reference to `cblas_sgemv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sger':

/tmp/go-build/cgo-gcc-prolog:1991: undefined reference to `cblas_sger'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_snrm2':

/tmp/go-build/cgo-gcc-prolog:2011: undefined reference to `cblas_snrm2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_srot':

/tmp/go-build/cgo-gcc-prolog:2035: undefined reference to `cblas_srot'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_srotm':

/tmp/go-build/cgo-gcc-prolog:2070: undefined reference to `cblas_srotm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_srotmg':

/tmp/go-build/cgo-gcc-prolog:2087: undefined reference to `cblas_srotmg'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ssbmv':

/tmp/go-build/cgo-gcc-prolog:2113: undefined reference to `cblas_ssbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sspmv':

/tmp/go-build/cgo-gcc-prolog:2151: undefined reference to `cblas_sspmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sspr':

/tmp/go-build/cgo-gcc-prolog:2170: undefined reference to `cblas_sspr'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sspr2':

/tmp/go-build/cgo-gcc-prolog:2192: undefined reference to `cblas_sspr2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sswap':

/tmp/go-build/cgo-gcc-prolog:2211: undefined reference to `cblas_sswap'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ssymm':

/tmp/go-build/cgo-gcc-prolog:2237: undefined reference to `cblas_ssymm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ssymv':

/tmp/go-build/cgo-gcc-prolog:2261: undefined reference to `cblas_ssymv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ssyr':

/tmp/go-build/cgo-gcc-prolog:2282: undefined reference to `cblas_ssyr'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ssyr2':

/tmp/go-build/cgo-gcc-prolog:2306: undefined reference to `cblas_ssyr2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ssyr2k':

/tmp/go-build/cgo-gcc-prolog:2332: undefined reference to `cblas_ssyr2k'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ssyrk':

/tmp/go-build/cgo-gcc-prolog:2355: undefined reference to `cblas_ssyrk'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_stbmv':

/tmp/go-build/cgo-gcc-prolog:2378: undefined reference to `cblas_stbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_stbsv':

/tmp/go-build/cgo-gcc-prolog:2401: undefined reference to `cblas_stbsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_stpmv':

/tmp/go-build/cgo-gcc-prolog:2422: undefined reference to `cblas_stpmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_stpsv':

/tmp/go-build/cgo-gcc-prolog:2443: undefined reference to `cblas_stpsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_strmm':

/tmp/go-build/cgo-gcc-prolog:2468: undefined reference to `cblas_strmm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_strmv':

/tmp/go-build/cgo-gcc-prolog:2491: undefined reference to `cblas_strmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_strsm':

/tmp/go-build/cgo-gcc-prolog:2516: undefined reference to `cblas_strsm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_strsv':

/tmp/go-build/cgo-gcc-prolog:2539: undefined reference to `cblas_strsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zaxpy':

/tmp/go-build/cgo-gcc-prolog:2559: undefined reference to `cblas_zaxpy'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zcopy':

/tmp/go-build/cgo-gcc-prolog:2578: undefined reference to `cblas_zcopy'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zdotc_sub':

/tmp/go-build/cgo-gcc-prolog:2598: undefined reference to `cblas_zdotc_sub'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zdotu_sub':

/tmp/go-build/cgo-gcc-prolog:2618: undefined reference to `cblas_zdotu_sub'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zgbmv':

/tmp/go-build/cgo-gcc-prolog:2663: undefined reference to `cblas_zgbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zgemm':

/tmp/go-build/cgo-gcc-prolog:2691: undefined reference to `cblas_zgemm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zgemv':

/tmp/go-build/cgo-gcc-prolog:2717: undefined reference to `cblas_zgemv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zgerc':

/tmp/go-build/cgo-gcc-prolog:2742: undefined reference to `cblas_zgerc'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zgeru':

/tmp/go-build/cgo-gcc-prolog:2767: undefined reference to `cblas_zgeru'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zhbmv':

/tmp/go-build/cgo-gcc-prolog:2793: undefined reference to `cblas_zhbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zhemm':

/tmp/go-build/cgo-gcc-prolog:2821: undefined reference to `cblas_zhemm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zhemv':

/tmp/go-build/cgo-gcc-prolog:2847: undefined reference to `cblas_zhemv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zher':

/tmp/go-build/cgo-gcc-prolog:2869: undefined reference to `cblas_zher'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zher2':

/tmp/go-build/cgo-gcc-prolog:2894: undefined reference to `cblas_zher2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zher2k':

/tmp/go-build/cgo-gcc-prolog:2922: undefined reference to `cblas_zher2k'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zherk':

/tmp/go-build/cgo-gcc-prolog:2947: undefined reference to `cblas_zherk'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zhpmv':

/tmp/go-build/cgo-gcc-prolog:2971: undefined reference to `cblas_zhpmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zhpr':

/tmp/go-build/cgo-gcc-prolog:2991: undefined reference to `cblas_zhpr'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zhpr2':

/tmp/go-build/cgo-gcc-prolog:3014: undefined reference to `cblas_zhpr2'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zswap':

/tmp/go-build/cgo-gcc-prolog:3050: undefined reference to `cblas_zswap'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zsymm':

/tmp/go-build/cgo-gcc-prolog:3078: undefined reference to `cblas_zsymm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zsyr2k':

/tmp/go-build/cgo-gcc-prolog:3106: undefined reference to `cblas_zsyr2k'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zsyrk':

/tmp/go-build/cgo-gcc-prolog:3131: undefined reference to `cblas_zsyrk'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ztbmv':

/tmp/go-build/cgo-gcc-prolog:3154: undefined reference to `cblas_ztbmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ztbsv':

/tmp/go-build/cgo-gcc-prolog:3177: undefined reference to `cblas_ztbsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ztpmv':

/tmp/go-build/cgo-gcc-prolog:3198: undefined reference to `cblas_ztpmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ztpsv':

/tmp/go-build/cgo-gcc-prolog:3219: undefined reference to `cblas_ztpsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ztrmm':

/tmp/go-build/cgo-gcc-prolog:3245: undefined reference to `cblas_ztrmm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ztrmv':

/tmp/go-build/cgo-gcc-prolog:3268: undefined reference to `cblas_ztrmv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ztrsm':

/tmp/go-build/cgo-gcc-prolog:3294: undefined reference to `cblas_ztrsm'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_ztrsv':

/tmp/go-build/cgo-gcc-prolog:3317: undefined reference to `cblas_ztrsv'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_cscal':

/tmp/go-build/cgo-gcc-prolog:507: undefined reference to `cblas_cscal'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_csscal':

/tmp/go-build/cgo-gcc-prolog:523: undefined reference to `cblas_csscal'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_drotg':

/tmp/go-build/cgo-gcc-prolog:1060: undefined reference to `cblas_drotg'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_dscal':

/tmp/go-build/cgo-gcc-prolog:1139: undefined reference to `cblas_dscal'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_srotg':

/tmp/go-build/cgo-gcc-prolog:2050: undefined reference to `cblas_srotg'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_sscal':

/tmp/go-build/cgo-gcc-prolog:2129: undefined reference to `cblas_sscal'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zdscal':

/tmp/go-build/cgo-gcc-prolog:2635: undefined reference to `cblas_zdscal'

c:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: $WORK\b001\_x002.o: in function `_cgo_4e5d5a0575bb_Cfunc_cblas_zscal':

/tmp/go-build/cgo-gcc-prolog:3031: undefined reference to `cblas_zscal'

collect2.exe: error: ld returned 1 exit status
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         go-version: [1.16.x, 1.15.x]
-        platform: [ubuntu-latest]
+        platform: [ubuntu-latest, windows-latest]
 
     runs-on: ${{ matrix.platform }}
     env:
@@ -20,7 +20,7 @@ jobs:
         GODEBUG: cgocheck=0
     defaults:
         run:
-            working-directory: ${{ env.GOPATH }}/src/gonum.org/v1/gonum
+            working-directory: ${{ env.GOPATH }}/src/gonum.org/v1/hdf5
 
     steps:
     - name: Install Go
@@ -46,16 +46,39 @@ jobs:
       with:
           path: ${{ env.GOPATH }}/src/gonum.org/v1/hdf5
 
+    - name: Install Windows packages
+      if: matrix.platform == 'windows-latest'
+      uses: msys2/setup-msys2@v2
+      with:
+        update: true
+        install: mingw-w64-x86_64-toolchain mingw-w64-x86_64-hdf5
+        msystem: MINGW64
+        path-type: inherit
+
     - name: Install Linux packages
       if: matrix.platform == 'ubuntu-latest'
       run: |
         sudo apt-get update
         sudo apt-get install -qq pkg-config libhdf5-serial-dev
 
-    - name: Build
+    - name: Build Windows
+      if: matrix.platform == 'windows-latest'
+      run: |
+        go install ./...
+      shell: msys2 {0}
+
+    - name: Build Linux
+      if: matrix.platform == 'ubuntu-latest'
       run: |
         go install ./...
 
-    - name: Test
+    - name: Test Windows
+      if: matrix.platform == 'windows-latest'
+      run: |
+        go test ./...
+      shell: msys2 {0}
+
+    - name: Test Linux
+      if: matrix.platform == 'ubuntu-latest'
       run: |
         go test ./...

--- a/h5a_attribute.go
+++ b/h5a_attribute.go
@@ -93,7 +93,7 @@ func (s *Attribute) Read(data interface{}, dtype *Datatype) error {
 
 		dtype = dtAttr
 		dlen := dtype.Size()
-		cstr := (*C.char)(unsafe.Pointer(C.malloc(C.ulong(uint(unsafe.Sizeof(byte(0))) * (dlen + 1)))))
+		cstr := (*C.char)(unsafe.Pointer(C.malloc(C.size_t(uint(unsafe.Sizeof(byte(0))) * (dlen + 1)))))
 		defer C.free(unsafe.Pointer(cstr))
 		switch {
 		case C.H5Tis_variable_str(dtAttr.Identifier.id) != 0:

--- a/h5d_dataset_test.go
+++ b/h5d_dataset_test.go
@@ -30,12 +30,14 @@ func createDataset1(t *testing.T) error {
 		t.Fatal(err)
 		return err
 	}
+	defer dspace.Close()
 
 	dset, err := f.CreateDataset("dset", T_NATIVE_USHORT, dspace)
 	if err != nil {
 		t.Fatal(err)
 		return err
 	}
+	defer dset.Close()
 
 	err = dset.Write(&data[0])
 	if err != nil {
@@ -77,9 +79,11 @@ func TestReadSubset(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer dset.Close()
 
 	// get the filespace and select the subset
 	filespace := dset.Space()
+	defer filespace.Close()
 	offset, stride, count, block := [2]uint{5, 1}, [2]uint{1, 1}, [2]uint{5, 2}, [2]uint{1, 1}
 	err = filespace.SelectHyperslab(offset[:], stride[:], count[:], block[:])
 	if err != nil {
@@ -95,6 +99,7 @@ func TestReadSubset(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer memspace.Close()
 
 	expected := [10]uint16{26, 27, 31, 32, 36, 37, 41, 42, 46, 47}
 
@@ -141,11 +146,13 @@ func TestWriteSubset(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer fspace.Close()
 	mdims := []uint{2, 6}
 	mspace, err := CreateSimpleDataspace(mdims, nil)
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer mspace.Close()
 
 	f, err := CreateFile(fname, F_ACC_TRUNC)
 	if err != nil {
@@ -157,6 +164,7 @@ func TestWriteSubset(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer dset.Close()
 
 	offset := []uint{6, 0, 0}
 	stride := []uint{3, 1, 1}
@@ -175,12 +183,14 @@ func TestWriteSubset(t *testing.T) {
 func TestSelectHyperslab(t *testing.T) {
 	DisplayErrors(true)
 	defer DisplayErrors(false)
+	defer os.Remove(fname)
 
 	dims := []uint{12, 4}
 	dspace, err := CreateSimpleDataspace(dims, nil)
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer dspace.Close()
 	offset, stride, count, block := []uint{1, 2}, []uint{2, 1}, []uint{4, 2}, []uint{1, 1}
 	if err = dspace.SelectHyperslab(offset, stride, count, block); err != nil {
 		t.Fatal(err)

--- a/h5f_file_test.go
+++ b/h5f_file_test.go
@@ -41,6 +41,7 @@ func TestFile(t *testing.T) {
 	}
 
 	f2 := f.File()
+	defer f2.Close()
 	fName := f.FileName()
 	f2Name := f2.FileName()
 	if fName != f2Name {
@@ -53,6 +54,7 @@ func TestFile(t *testing.T) {
 	if err != nil {
 		t.Fatalf("CreateGroup() failed: %s", err)
 	}
+	defer g.Close()
 	if name := g.Name(); name != "/"+groupName {
 		t.Fatalf("Group Name() have %v, want /%v", name, groupName)
 	}
@@ -61,6 +63,7 @@ func TestFile(t *testing.T) {
 	if err != nil {
 		t.Fatalf("OpenGroup() failed: %s", err)
 	}
+	defer g2.Close()
 	if name := g2.Name(); name != "/"+groupName {
 		t.Fatalf("Group Name() have %v, want /%v", name, groupName)
 	}
@@ -85,11 +88,13 @@ func TestFile(t *testing.T) {
 	if err != nil {
 		t.Fatalf("CreateDataspace failed: %s", err)
 	}
+	defer ds.Close()
 	dsetName := "test_dataset"
 	dset, err := f.CreateDataset(dsetName, T_NATIVE_INT, ds)
 	if err != nil {
 		t.Fatalf("CreateDataset failed: %s", err)
 	}
+	defer dset.Close()
 	if name := dset.Name(); name != "/"+dsetName {
 		t.Fatalf("Dataset Name() have %v, want /%v", name, dsetName)
 	}
@@ -97,6 +102,7 @@ func TestFile(t *testing.T) {
 	if dFile.Name() != f.Name() {
 		t.Fatalf("Dataset File() have %v, want %v", dFile.Name(), f.Name())
 	}
+	defer dFile.Close()
 
 	if n, err := f.NumObjects(); err != nil {
 		t.Fatalf("NumObjects failed: %s", err)

--- a/h5g_group_test.go
+++ b/h5g_group_test.go
@@ -25,7 +25,9 @@ func TestGroup(t *testing.T) {
 	if err != nil {
 		t.Fatalf("couldn't create group: %s", err)
 	}
-	if *g1.File() != *f {
+	g1f := g1.File()
+	defer g1f.Close()
+	if *g1f != *f {
 		t.Fatal("wrong file for group")
 	}
 	if g1.Name() != "/foo" {
@@ -44,7 +46,9 @@ func TestGroup(t *testing.T) {
 	if err != nil {
 		t.Fatalf("couldn't create group: %s", err)
 	}
-	if *g2.File() != *f {
+	g2f := g2.File()
+	defer g2f.Close()
+	if *g2f != *f {
 		t.Fatal("wrong file for group")
 	}
 	if g2.Name() != "/foo/bar" {
@@ -58,7 +62,9 @@ func TestGroup(t *testing.T) {
 	if err != nil {
 		t.Fatalf("couldn't create group: %s", err)
 	}
-	if *g3.File() != *f {
+	g3f := g3.File()
+	defer g3f.Close()
+	if *g3f != *f {
 		t.Fatal("wrong file for group")
 	}
 	if g3.Name() != "/foo/bar/baz" {
@@ -100,11 +106,13 @@ func TestGroup(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer g2.Close()
 
 	g3, err = g2.OpenGroup("baz")
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer g3.Close()
 
 	_, err = g3.OpenGroup("bs")
 	if err == nil {
@@ -117,22 +125,26 @@ func TestGroup(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer dtype.Close()
 
 	dims := []uint{1}
 	dspace, err := CreateSimpleDataspace(dims, dims)
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer dspace.Close()
 
 	dset, err := g3.CreateDataset("dset", dtype, dspace)
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer dset.Close()
 
 	dset2, err := g3.OpenDataset("dset")
 	if dset.Name() != dset2.Name() {
 		t.Error("expected dataset names to be equal")
 	}
+	defer dset2.Close()
 
 	dset2, err = g3.OpenDataset("bs")
 	if err == nil {

--- a/h5p_proplist_test.go
+++ b/h5p_proplist_test.go
@@ -184,6 +184,7 @@ func save(fn, dsn string, dims []uint, dcpl *PropList) ([]float64, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer dspace.Close()
 
 	dset, err := f.CreateDatasetWith(dsn, T_NATIVE_DOUBLE, dspace, dcpl)
 	if err != nil {

